docs/concepts/trading-on-vega/positions-margin.md

```diff
@@ -75,6 +74,12 @@ The amount a trader will have held aside as maintenance margin is derived from t
 
 If the margin balance drops below the maintenance margin level, the position closeout process gets initiated.
 
+For [perpetual futures markets](./market-types.md#perpetual-futures), the margin calculations contain additional term to capture the exposure of a given position to an upcoming funding payment. The market proposal includes a margin funding factor that determines to what degree the funding payment amount impacts a trader's maintenance margin.
```

Comment:
```suggestion
For [perpetual futures markets](./market-types.md#perpetual-futures), the margin calculations contain additional term to capture the exposure of a given position to an upcoming funding payment. The market proposal includes a margin funding factor that determines to what degree the funding payment amount impacts a trader's maintenance margin. This additional term can only increase the margin requirement if a given position is expected to make a payment at the end of the current funding period, but it will never decrease the margin requirement, even if party is expecting to receive a funding payment.
```